common/src/main/scala/services/vehicle/support/VehicleRemover.scala

```diff
@@ -26,7 +27,9 @@ class VehicleRemover extends RemoverActor {
     entry.zone.GUID(vehicleGUID) match {
       case Some(vehicle : Vehicle) if vehicle.HasGUID =>
         val zoneId = entry.zone.Id
-        vehicle.Actor ! Vehicle.PrepareForDeletion()
+        if(vehicle.Actor != ActorRef.noSender) {
+          vehicle.Actor ! Vehicle.PrepareForDeletion()
+        }
```

Comment:
I'll allow it, though the bulk of this file will be going out the window.

Reply:
Yeah I figured when I saw you were getting rid of VehicleRemover, but at least it should work as a temporary bandaid